### DISCUSSION TITLE
Unable to upload proof of enrollment for student account, webpage not accessing webcam

### DISCUSSION THREAD

**Discussion by Septicswan** [community/community#46573] [GitHub Education]
Hi All, 

I am trying to sign up for student access and I'm at the point where they would like me to upload proof of my enrollment, which is where I'm encountering my problem. For some reason the webpage (despite permissions being granted) will not access my webcam to take a picture, I just get the arrows spinning endlessly. I've tried reloading the page a few times but there is never any progress. 

Is there a support email where I can email my proof of enrollment? This isn't anything absolutely critical, however if I am able to get the benefit of a student account, why not? 

Thanks for your help!

**Comment by jge162** [comment]
You don’t have to use webcam. You can take a picture with your smart phone. Email it to yourself and upload it too.

**Reply by Septicswan** [reply]
![image](https://user-images.githubusercontent.com/124642799/217128354-d2a1fb96-779f-4fe8-85de-7dae5c0177f0.png)
There doesn't seem to be an option to just 'upload' something (which is the main reason why I posted this question)

**Reply by jge162** [reply]
When I submitted my approval I uploaded my picture. Maybe they have changed it. I’ll login and double check.

**Reply by jge162** [reply]
Well I don’t get to see that option to upload because I’m already approved. Try reaching out to education support. They can assist you further. Here is the [Link](https://support.github.com/request/education?tags=hubberfy_education)

**Reply by Septicswan** [reply]
Thanks for your help!
